src/NRedisStack/Json/IJsonCommandsAsync.cs

```diff
@@ -157,7 +157,7 @@ public interface IJsonCommandsAsync
     /// <param name="value">The value to increment by.</param>
     /// <returns>The new values after being incremented, or null if the path resolved a non-numeric.</returns>
     /// <remarks><seealso href="https://redis.io/commands/json.numincrby"/></remarks>
-    Task<double?[]> NumIncrbyAsync(RedisKey key, string path, double value);
+    Task<double?[]> NumIncrByAsync(RedisKey key, string path, double value);
```

Comment:
1. Its breaking change
2. The sync command still called `NumIncrby`

Reply:
Thank you for the feedback you gave me and the time you took to review
This is valuable for me
Actually, when I applied these changes, it was after work and I was very tired
  I was reviewing the codes of this project out of curiosity and made some changes
To be honest, I am very interested in participating and contributing to this project
I will fix the problems you mentioned and I would be grateful if you check and give me feedback again
I can fix all the parts you mentioned
We can also communicate via email or wherever you prefer

Reply:
Hi Nima, I pushed a small commit reverting these `ToString()` calls. It was easiest, given that it's a super small change. Thanks for your patience!